lib/pcm_stream_channel_modifier/pcm_stream_channel_modifier.c

```diff
@@ -10,8 +10,10 @@
 #include <errno.h>
 
 #include <zephyr/logging/log.h>
+#include <stdlib.h>
```

Comment:
There should IMO be no changes to this file. We need to look for another place to put this functionality. This is a NCS global file and should only do the exact thing it says in the doc an nothing more. Unit tests are also required for changes here.